rules/composer/src/ComposerModifier.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Composer;
+
+use Rector\Composer\Contract\Rector\ComposerRectorInterface;
+use Symplify\ComposerJsonManipulator\ValueObject\ComposerJson;
+
+final class ComposerModifier
+{
+    /**
+     * @var string
+     */
+    private const COMPOSER_UPDATE = 'composer update';
```

Comment:
There is no need to have it here with getter if it is constant. In my code it was possible to change the executed command via setter - if somebody need to run it with parameters or use composer.phar etc

Reply:
It was used only in tests. This should be configured on parameter level and documented, not on service setter level.